Chainlit/chainlit
# Auth0 inspired UX

https://github.com/Chainlit/chainlit/assets/10745953/a1b6db89-3b2b-469f-bb06-fb962249ac1b

⚠️ The issue during signin is fixed on the be

## Changes

- Use `react-password-checklist` to display password validation
- Reset error display on switching form
- Fix display of microsoft provider button

## How to test?

- Go to signup form
- Notice the password creation flow
- Notice that you can still login normally
